library/src/main/scala/scala/scalajs/js/WrappedDictionary.scala

```diff
@@ -57,7 +57,7 @@ class WrappedDictionary[A](val dict: Dictionary[A])
   }
 
   override def update(key: String, value: A): Unit =
```

Comment:
No, the class itself is `@inline`. If the instance of `WrappedDictionary` was not virtualized away, there's no point in inlining `update`. But if it is, then `update` will be inlined automatically.